test/specs/scroll-transition.test.js

```diff
@@ -53,4 +53,92 @@ describe('<ScrollTransition />', () => {
       expect(instance.getSlideDirection(start, end, isWrapping)).toEqual(-1);
     });
   });
+
+  describe('#getSlideStyles', () => {
+    describe('calculating zoomScale', () => {
+      it('should return zoomScale of 1 for all slides when animation !== zoom , ', () => {
+        const wrapper = mount(
+          <ScrollTransition>
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(1)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and 0.85 as a default for slide index 1 & 2 when animation === zoom but no zoomScale defined ', () => {
+        const wrapper = mount(
+          <ScrollTransition animation="zoom">
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(0.85)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(0.85)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and zoomScale of 0.1 for index 1 & 2 when animation === zoom and zoomScale = 0.1 ', () => {
```

Comment:
I also think you can be more generic in your wording for your it statements.  

Instead of saying:
"should ... for slide index 0 and ... for index 1 & 2 ..."

You could say:
"should ... for the _current slide_ and ... for the _remaining slides_ ..."

This way if someone is just reading through the tests to see what this prop does, it's clear that it will generically work against any amount of slides and will apply different zoomScales for the `current slide` and the `remaining slides` and not just slide index 0, 1, or 2.

Reply:
Fixed